transferui/src/test/java/com/hyperwallet/android/ui/transfer/viewmodel/ListTransferDestinationViewModelTest.java

```diff
@@ -274,13 +277,22 @@ public Object answer(InvocationOnMock invocation) {
                 callback.onError(errors);
                 return callback;
             }
-        }).when(repository).loadTransferMethods(ArgumentMatchers.any(
+        }).when(mTransferMethodRepository).loadTransferMethods(ArgumentMatchers.any(
                 TransferMethodRepository.LoadTransferMethodListCallback.class));
 
         ListTransferDestinationViewModel viewModel = factory.create(ListTransferDestinationViewModel.class);
+        viewModel.init();
 
         assertThat(viewModel.getSelectedTransferDestination().getValue(), is(nullValue()));
         assertThat(viewModel.getTransferDestinationList().getValue(), is(nullValue()));
         assertThat(viewModel.isLoading().getValue(), is(false));
     }
+
+
+    @Test
+    public void testInit() {
+        ListTransferDestinationViewModel viewModel = spy(mListTransferDestinationViewModel);
+        viewModel.init();
+        verify(viewModel, times(1)).loadTransferDestinationList();
```

Comment:
verify defaults to execute once, no need to explicitly specify

Reply:
Done